src/main/kotlin/no/nav/pawproxy/app/Environment.kt

```diff
@@ -20,18 +20,3 @@ fun requireClusterName() =
 
 fun requireNamespace(): String =
     requireProperty("NAIS_NAMESPACE")
-
```

Comment:
Liker at du har fjernet en del av dette, og heller benytter ulik config for de ulike miljøene. Tror det bidrar til å ta ned kompleksiteten i koden.

Reply:
Ja, jeg fant ut at det ikke var i bruk, så ryddet det bort :)